src/integrationTest/resources/application-integration-test.yml

```diff
@@ -17,6 +17,24 @@ spring:
         jwt:
           public-key-location: classpath:local-public-key.pub
 
+hmpps.sqs:
+  enabled: true
+  provider: localstack
+  queues:
+    educationandworkplan:
+      queueName: education-and-work-plan-queue
+      subscribeFilter: "{\"eventType\":[\"prison-offender-events.prisoner.received\", \"prison-offender-events.prisoner.released\"]}"
```

Comment:
You may want to check you are listening to the right release event.  There are actually two (domain) events from different sources.

I think you probably want this want this `prisoner-offender-search.prisoner.released`???

The one you are using here is delayed in production (I think by approx 45 mins).  Perhaps double check with the Syscon team?

Reply:
Yeah, this came up in a slack conversation with Andy Lee and Andy Marke: https://mojdt.slack.com/archives/C69NWE339/p1722937869998369

They mentioned the delay factor as well.

As I understand the purpose of the delay, and our use case for want to know when people are released, I _think_ this is the right one to use. (We want to know when they really have been released, rather than potentially having to deal with a release then admission event in fairly quick succession.

Unless I've misunderstood what the delay is for? (it's quite possible! 🤣 )